lib/include/irritator/examples.hpp

```diff
@@ -320,6 +320,260 @@ example_qss_negative_lif(simulation& sim, F f) noexcept
     return status::success;
 }
 
+/** @brief Implements the Linear SEIR model for QSS1, QSS2 and QSS3.
+     *
+     * @details
+     *
+     * @Tparam QssLevel
+     * @Tparam F
+     * @param sim
+     * @param f
+     * @return
+     */
+
+    template<int QssLevel, typename F>
+    status
+        example_qss_seir_lineaire(simulation& sim, F f) noexcept
+    {
+        static_assert(1 <= QssLevel && QssLevel <= 3, "Only for Qss1, 2 and 3");
+
+        bool success = sim.can_alloc<abstract_wsum<QssLevel, 2>>(2) &&                                    
```

Comment:
Please, this is an old API. Can you update your repository and use:
`sim.models.can_alloc(10)`